<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1246</width>
    <height>796</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>110</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sort</string>
    </property>
   </widget>
   <widget class="QPushButton" name="filterSortBtn">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>220</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>250</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>280</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Type of skin</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>120</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Price 1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sortLineEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>150</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="typeOfSkinAddLineEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>280</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>351</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Display Product</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameRemoveLineEdit">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>380</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>160</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>400</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Benefit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addNewProductBtn">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Add New Product</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>120</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Type</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="price2AddLineEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>200</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saleProductBtn">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>190</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sale Product</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameSaleLineEdit">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>250</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>160</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Price 1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="amountAddLineEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>240</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addOldProductBtn">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>10</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Add Old Product</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="price1AddOldLineEdit">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>120</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="productListWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>461</width>
      <height>451</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>380</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="benefitAddLineEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>400</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="colorAddLineEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>360</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="amountAddOldLineEdit">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>160</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="amountSaleLineEdit">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>290</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>40</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Filter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameAddOldLineEdit">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>80</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="price1AddLineEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>160</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="typeAddLineEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>120</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="aromaAddLineEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>320</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="removeProductBtn">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>320</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Remove Product</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameAddLineEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>80</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>360</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Color</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>200</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Price 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filterLineEdit">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>80</y>
      <width>113</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>80</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>290</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>320</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Aroma</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>240</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>80</y>
      <width>49</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1246</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
